SIRShiny.rmd

```diff
@@ -0,0 +1,159 @@
+library(shiny)
+library(dplyr)
+library(ggplot2)
+
+# Define UI for application that draws a histogram
+ui <- fluidPage(
+  
+  # Application title
+  titlePanel("SIR Modeling - Britney, Linh, and Bowen"),
+  
+  # Sidebar with a slider input for number of bins 
+  sidebarLayout(
+    sidebarPanel(
+      checkboxGroupInput("Random",
+                         "Variability?",
+                         "Yes"),
+      sliderInput("Pop",
+                  "Population of our town",
+                  min = 50,
+                  max = 2000,
+                  value = 500),
+      sliderInput("Days",
+                   "Number of Days you want to see (Select -1 to see when the disease will go away)",
+                   min = -1,
+                   max = 200,
+                   step = 1, 
+                   value = 50),
+      sliderInput("Initialinf",
+                   "Initial People infected",
+                  min = 1,
+                  max = 100,
+                  step = 1, 
+                  value = 5),
+      sliderInput("Spread_Rate",
+                  "Spread Rate", 
+                  min = 0.01, 
+                  max = 1.00, 
+                  step = 0.01,
+                  value = 0.01),
+      sliderInput("Treat_Rate",
+                  "Treatment Rate", 
+                  min = 0.01, 
```

Comment:
Since we are not doing any treatment here, I think we should make it as the recovery rate